test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -501,6 +502,59 @@ func testResourceDelete(c *testContext) {
 		c.t.Error(err)
 		return
 	}
+
```

Comment:
Call `c.admissionHolder.verifyResult(c.t)` and then set another expectation if you want to test multiple delete calls are made and passed to admission